person.rb

```diff
@@ -1,11 +1,14 @@
 require './nameable'
 class Person < Nameable
+  attr_reader :rentals
```

Comment:
- [ ] The `:age`, `:name`, and `:rentals` should be readable and writable, kindly make sure to add them to `attr_accessor` so it is accessible (read/write) by the class